tmp/build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -0,0 +1,212 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: maistra.v__VERSION__
+  namespace: placeholder
+  annotations:
+    categories: "OpenShift Optional, Integration & Delivery"
+    capabilities: "Seamless Upgrades"
+    certified: "false"
+    repository: https://github.com/maistra/istio-operator
+    description: |-
+      The Service Mesh Operator provides means for configuring and managing an instance of Red Hat OpenShift Service Mesh.
+      ## Prerequisites and Requirements
+      ### Maistra Operator Namespace
+
+    containerImage: __IMAGE_SRC__
+    createdAt: 2019-07-18T08:00:00Z
+    support: Red Hat, Inc. 
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "maistra.io/v1",
+          "kind": "ServiceMeshControlPlane",
+          "metadata": {
+            "name": "basic-install"
+          },
+          "spec": {
+            "istio": {
+              "global": {
+                "proxy": {
+                  "resources": {
+                    "requests": {
+                      "cpu": "100m",
+                      "memory": "128Mi"
+                    },
+                    "limits": {
+                      "cpu": "500m",
+                      "memory": "128Mi"
+                    }
+                  }
+                }
+              },
+              "gateways": {
+                "istio-egressgateway": {
+                  "autoscaleEnabled": false
+                },
+                "istio-ingressgateway": {
+                  "autoscaleEnabled": false,
+                  "ior_enabled": false
+                }
+              },
+              "mixer": {
+                "policy": {
+                  "autoscaleEnabled": false
+                },
+                "telemetry": {
+                  "autoscaleEnabled": false,
+                  "resources": {
+                    "requests": {
+                      "cpu": "100m",
+                      "memory": "1G"
+                    },
+                    "limits": {
+                      "cpu": "500m",
+                      "memory": "4G"
+                    }
+                  }
+                }
+              },
+              "pilot": {
+                "autoscaleEnabled": false,
+                "traceSampling": 100.0
+              },
+              "kiali": {
+                "enabled": true,
+                "dashboard": {
```

Comment:
Worth to remove this dashboard section to keep in sync with our examples?

Reply:
Updated the example to match our docs.